core-bundle/tests/Routing/RouteProviderTest.php

```diff
@@ -664,14 +673,15 @@ private function createPage(string $language, string $alias, bool $fallback = tr
     {
         /** @var PageModel&MockObject $page */
         $page = $this->mockClassWithProperties(PageModel::class);
-        $page->id = random_int(1, 10000);
+        $page->id = ++$this->pageModelAutoIncrement;
```

Comment:
@aschempp Is it OK to just count upwards here? I’m not sure how this can or should affect the test results.